### DISCUSSION TITLE
Task --> Identify only those scenes that are not organized

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#5491] [Q&A] [answered]
How to identify only those scenes that are not organized (from Tasks)? (i.e. Scenes without Organized flag)

![image](https://github.com/user-attachments/assets/1ff2a685-b664-48d2-b45d-c658a59f39bc)

**Comment by DogmaDragon** [comment, marked as answer]
Those are the only ones that will be identified. Organized flag automatically excludes marked objects from automatic tasks.
